### PR TITLE
Add Atlas init action

### PR DESCRIPTION
**Pull request by roderickyao**
Thanks @DariuszAniszewski contributing to this init action. Upgraded with Atlas 1.2. This init action now should be used together with Dataproc 1.5+

### CODE REVIEW

**Comment by roderickyao** [comment]
d any issues), please reply here with

**Comment by roderickyao** [comment]
@googlebot I signed it!

File: atlas/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* HBase
+* Solr
+
+The init action will fail if any of those services is not available.
+
+Use the following command to create standard Dataproc cluster with Atlas:
+```
+gcloud dataproc clusters create <CLUSTER_NAME> \
+    --optional-component ZOOKEEPER \
+    --initialization-actions 'gs://dataproc-initialization-actions/hbase/hbase.sh','gs://dataproc-initialization-actions/solr/solr.sh',"gs://dataproc-initialization-actions/atlas/atlas.sh" \
```
Comment (medb) [inline atlas/README.md:25]: Use `SOLR` and `HBASE` components instead?

File: atlas/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# There is no pre-build tarball available if not using Dataproc 1.5+
+# Following commands can be used to build Atlas 1.2.0 from source
+
+# install maven 3.6
+wget https://archive.apache.org/dist/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.tar.gz -P /opt
```
Comment (medb) [inline atlas/build.sh:6]: Dataproc has Atlas in APT repo, may you use it instead: `apt-get install atlas`?

Comment (roderickyao) [reply atlas/build.sh]: The thoughts were to use this to allow customer build. But since let's only enable this for 1.5. Removed this build script.

**Comment by medb** [comment]
Closing in favor of original https://github.com/GoogleCloudDataproc/initialization-actions/pull/478 PR

**Pull request closed without merging**